pythonforandroid/recipes/kivy/__init__.py

```diff
@@ -34,6 +34,13 @@ class KivyRecipe(PyProjectRecipe):
     # WARNING: Remove this patch when a new Kivy version is released.
     patches = [("sdl-gl-swapwindow-nogil.patch", is_kivy_affected_by_deadlock_issue), "use_cython.patch"]
 
+    @property
+    def need_stl_shared(self):
+        if "sdl3" in self.ctx.recipe_build_order:
+            return True
+        else:
+            return False
```

Comment:
late post merge review, why not simply a one liner `return "sdl3" in self.ctx.recipe_build_order`